changelog/17009.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+UI: Added JWT authentication warning message about blocked pop-up windows and web browser settings.
```

Comment:
Looking at other changelog entries it seems the descriptor before the colon is lowercase. Would you mind updating UI to lowercase while you're in there?

Reply:
Corrected - thanks again @zofskeez